nucliadb/tests/search/integration/api/v1/test_suggest.py

```diff
@@ -47,15 +47,14 @@ async def test_suggest_resource_all(
     driver = get_driver()
     async with driver.transaction(read_only=True) as txn:
         key = KB_SHARDS.format(kbid=kbid)
+        node_obj = get_nidx_fake_node()
         async for key in txn.keys(key):
             value = await txn.get(key)
             assert value is not None
             shards = PBShards()
             shards.ParseFromString(value)
             for replica in shards.shards[0].replicas:
-                node_obj = INDEX_NODES.get(replica.node)
-
-                if node_obj is not None:
+                if True:
```

Comment:
I think you can remove this loop (`for replica in shards.shards[0].replicas:`) entirely. Shard replicas will always be empty now. We probably want to keep the asserts inside, so maybe replace the loop by a single query to nidx fake node.